roles/motd/tasks/main.yml

```
@@ -0,0 +1,42 @@
+---
+- name: Create motd fragments directory
+  file:
```

Please make these fully qualified.

```suggestion
  ansible.builtin.file:
```